apps/block_scout_web/priv/gettext/default.pot

```diff
@@ -1967,3 +1955,25 @@ msgstr ""
 #: lib/block_scout_web/templates/block/overview.html.eex:58
 msgid "Round number"
 msgstr ""
+
+#, elixir-format
+#: lib/block_scout_web/templates/block/_tile.html.eex:39
+#: lib/block_scout_web/templates/block/overview.html.eex:135
+#: lib/block_scout_web/templates/chain/_block.html.eex:15
+msgid "Validator"
+msgstr ""
+
+#, elixir-format
+#: lib/block_scout_web/templates/api_docs/eth_rpc.html.eex:7
+msgid "This API is provided to support some rpc methods in the exact format specified for Celo nodes, which can be found "
+msgstr ""
+
+# From templates/transaction/_token_transfer.html.eex#L29
+# From templates/transaction/overview.html.eex#L245
+# From templates/transaction_token_transfer/_token_transfer.html.eex#L23
+msgid "cGLD"
+msgstr "CELO"
+
+# From templates/tokens/overview/details.html.eex#L8
+msgid "Celo Gold"
+msgstr "Celo"
```

Comment:
Shouldn't it be `CELO`? I would double check. I think `CELO` is the name of the currency, `Celo` is the name of the platform.